### ISSUE TITLE
Patchwork could not load this mod (Crash)

### ISSUE THREAD

**Issue by Sshmoob** [PatchworkMC/patchwork-patcher#98]
When I tested patchwork, everything worked fine except when I actually opened Minecraft with the mods installed. Minecraft would start, but then eventually crash. It said something along the lines of: "Patchwork could not load this mod correctly" or something like that, in the crash report. I don't know if this is a bug or if I did something wrong when setting it up. Also, I got that version of Patchwork from a YouTube video, and I think it was version 0.3. The mod I converted was the travelers backpack mod, which converted just fine, the problems only happened when I tried to open Minecraft with it, fabric API, and Patchwork API. when I was converting I didn't put anything in the data file before I ran patchwork, because the YouTube video didn't show to put anything in there. If I was supposed to put something in there, you can tell me.

**Comment by IMS212** [comment]
Taken from the patchwork discord
`- Can I use it?
The short answer is: No.
The long answer:
- Patchwork only runs a few mods (as of writing only a few mods )
- Patchwork is not looking for testers, and we do not want people who are not writing code for Patchwork using the software.
- As such, nobody is required to help you run Patcher or use Patchwork API with your favorite mods.
With that being said,
- You can always open bug reports on our github pages
- Everyone is encouraged to ask questions in the Development category about writing and using Patchwork's code, or asking about how Patchwork works.
- This note isn't a restriction on giving help, but instead a warning that you shouldn't expect official support.`
TL;DR: Patchwork is not done yet, and very few (only 2 known) mods work on 1.16. Whoever made that video is likely lying.
